Recommend eliciting an ACK soon after idle (#218)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

buffered to be sent, a sender can send an IMMEDIATE_ACK frame with the last data
packet before an idle period to avoid waiting for the ack delay.

> When a sender when there are no inflight packets, no acknowledgements will be

Reviewer: This sentence doesn't parse.

> When a sender when there are no inflight packets, no acknowledgements will be

Author: ```suggestion
When a sender has no inflight packets, it knows no acknowledgements will be
```

> When a sender when there are no inflight packets, no acknowledgements will be
> received for a single round trip in addition to whatever delay is caused by
> the Max Ack Delay and Ack-Eliciting Threshold values used by the receiver.

Author: ```suggestion
When resuming sending when there are no inflight packets, no acknowledgements will be
received for a single round trip.  Additionally, the Max Ack Delay and Ack-Eliciting Threshold values used by the receiver will further delay acknowledgements.
```

> values used by the receiver further delay acknowledgements.  In this case, the

Author: ```suggestion
values used by the receiver can further delay acknowledgements.  In this case, the
```

> received for a single round trip.  Additionally, the Max Ack Delay and Ack-Eliciting Threshold
> values used by the receiver will further delay acknowledgements.

Author: ```suggestion
received for a single round trip.  Additionally, the Max Ack Delay and Ack-Eliciting
Threshold values used by the receiver will further delay acknowledgements.
```

> It is RECOMMENDED the sender sends an IMMEDIATE_ACK or ACK_FREQUENCY frame in this
> case to avoid sending for substantally more than a round trip without any

Reviewer 2: ```suggestion
In this case the sender can send an IMMEDIATE_ACK or ACK_FREQUENCY frame together with the first data frame
to avoid sending for substantially more than a round trip without any
```

> for feedback.

Author: ```suggestion
for an acknowledgement.
```
